app/services/chasers/candidate/offer_email_service.rb

```diff
@@ -0,0 +1,10 @@
+module Chasers
+  module Candidate
+    class OfferEmailService
+      def self.call(application_choice:, mailer:, chaser_type:)
```

Comment:
It's not necessary because you are already doing the check in `OffersToChaseQuery`, but some of these services have a `self.already_sent_to?` method in case you want to follow that pattern.